### PR TITLE
Upload srpm + run copr (fix #159)

### LINKED ISSUE #159: make travis CI mock build rpm when commit is pushed
it should upload the result rpm somewhere to accessible storage

### PR DESCRIPTION
**Pull request by PavolVican**

### CODE REVIEW

**Comment by PavolVican** [comment]
tito not work correctly and not run mock
I have problem with travis encrypt variable, so you have to set manually Environment Variables in travis (settings in travis). I will send email with variable names and their values.

**Comment by jsilhan** [comment]
Great, now you can be the first fixing the flake8 violations :):
```
	E225 @ 4:14 - missing whitespace around operator

E501 @ 5:80 - line too long (115 > 79 characters)

E225 @ 8:27 - missing whitespace around operator

E225 @ 9:22 - missing whitespace around operator

E231 @ 12:57 - missing whitespace after ','

E231 @ 12:74 - missing whitespace after ','

E501 @ 12:80 - line too long (157 > 79 characters)

E231 @ 12:93 - missing whitespace after ','

E231 @ 12:110 - missing whitespace after ','

E231 @ 12:132 - missing whitespace after ','

E225 @ 13:10 - missing whitespace around operator

E231 @ 14:26 - missing whitespace after ','

W391 @ 15:1 - blank line at end of file
```
Can you also suppress the error loading srpm (exclude this file or force it to analyze just *.py files in folders not archives)

**Comment by PavolVican** [comment]
All Done!

**Comment by jsilhan** [comment]
Would it be possible to notify me about the CI error state when I merge it into the upstream not from PRs?

**Comment by jsilhan** [comment]
The CI seems to be working again in the master branch. Can you rebase it against the upstream?

**Comment by PavolVican** [comment]
OK... It is work.

File: rel-eng/travis/upload.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+for project in result:
+    if project.projectname == "rpg":
+        found_project = True
+        break
+if not found_project:
+    chroot = ["fedora-21-x86_64", "fedora-21-i386", "fedora-22-x86_64",
+              "fedora-22-i386", "fedora-rawhide-i386", "fedora-rawhide-x86_64"]
+    cl.create_project("rpg", chroots=chroot)
+srpm_name = 'https://github.com/PavolVican/rpg/raw/srpm/' + sys.argv[3]
+# cl.create_new_build("rpg", pkgs=[srpm_name])
```
Comment (jsilhan) [inline rel-eng/travis/upload.py]: why it's in comment?

**Comment by PavolVican** [comment]
I have forgotten to uncomment.

**Pull request merged**